examples/nextjs-example/src/components/UppyDashboard.tsx

```diff
@@ -0,0 +1,125 @@
+'use client'
+
+import Uppy from '@uppy/core'
+import { Dashboard } from '@uppy/react'
+import Transloadit from '@uppy/transloadit'
+import Tus from '@uppy/tus'
+import Xhr from '@uppy/xhr-upload'
+import { useState } from 'react'
+
+import '@uppy/core/css/style.min.css'
+import '@uppy/dashboard/css/style.min.css'
+import '@uppy/webcam/css/style.min.css'
+
+const UppyDashboard: React.FC = () => {
```

Comment:
React.FC is no longer recommend, we can just use regular functions and type the props separately.